tests/sources/test_clang_format.py

```diff
@@ -113,7 +113,7 @@ def test_sources_are_clang_format_compliant(self):
                     for fileName in fileNames:
                         extension = os.path.splitext(fileName)[1][1:].lower()
                         if extension in extensions:
-                            sources.append(os.path.join(rootPath, fileName))
+                            sources.append(os.path.join(rootPath, fileName).replace('/', os.sep))
```

Comment:
Are sure this is needed? It seems strange that this would be needed.

Reply:
It seems that without it clang-format doesn't work fine on appveyor:
> CalledProcessError: Command '['clang-format', '-style=file', u'projects/vehicles/plugins/robot_windows/automobile_window/OSMImportWidget.hpp']' returned non-zero exit status -1073741511

https://ci.appveyor.com/project/DavidMansolino/webots/builds/25195022

Reply:
Do you mean that `rootPath` is provided with `/` instead of `\` on Windows?

Reply:
It may be better to use `os.path.normpath()` as explained here: https://stackoverflow.com/questions/53481285/python-os-walk-displays-mixed-windows-and-unix-paths.